.github/actions/build-binaries/action.yaml

```diff
@@ -24,41 +24,61 @@ runs:
       run: ${{ inputs.build_command }}
       shell: bash
 
-    - name: Test Executable
+    - name: Set TEST_CLI_PATH
       run: |
-        ls -l dist
-        ./dist/${{ inputs.package_name }}/${{ inputs.package_name }}${{ inputs.operating_system == 'windows-latest' && '.exe' || '' }} --help
+        ls -la dist/algokit/
```

Comment:
Any reason to keep this?

Reply:
@neilcampbell was useful for debugging but not anymore, can always revert back if needed